src/helpers/articles/rateArticleHelper.js

```diff
@@ -0,0 +1,49 @@
+import db from '../../sequelize/models';
+
+const { Article } = db;
+
+/**
+ * @exports ArticRatelehelper
+ * @class Articlehelper
+ * @description Helps to generate token and passwor hashing
+ * */
+class ArticleRatelehelper {
+  /**
+       * Check the environment
+       * @function validateArticleRated
+       * @param  {integer} id - Check the rate
+       * @param  {integer} user - Check the user
+       * @return {string} Validate the rate
+       */
+  static async validateArticleRated(id, user) {
+    const findarticle = await Article.findOne({ where: { id } });
+    if (findarticle) {
+      const articleOwner = await Article.findAll({
+        where: { id, authorId: user }
+      });
+      if (articleOwner.length) {
+        return 'Sorry! You cannot rate your article';
+      }
+    }
+    if (findarticle == null) {
```

Comment:
Why don't you just use `else` here instead of `findarticle == null` since` else` in this particular instance is more accurate when it comes to evaluating the truthy and falsy conditions.

Reply:
@Malaba6  This won't work also.